moveit_core/trajectory_processing/src/ruckig_traj_smoothing.cpp

```diff
@@ -291,6 +291,7 @@ bool RuckigSmoothing::runRuckig(robot_trajectory::RobotTrajectory& trajectory,
       getNextRuckigInput(trajectory.getWayPointPtr(waypoint_idx), next_waypoint, group, ruckig_input);
 
       // Run Ruckig
+      ruckig::Trajectory<ruckig::DynamicDOFs> ruckig_trajectory;
```

Comment:
```suggestion
      ruckig::Trajectory<ruckig::DynamicDOFs> ruckig_trajectory(6);
```

I think the constructor requires the dof as constructor argument

Reply:
I got it to compile like so:

`ruckig::Trajectory<ruckig::DynamicDOFs, ruckig::StandardVector> ruckig_trajectory(ruckig::DynamicDOFs);`

Note sure if this will run well, yet.

Reply:
DynamicDOFs resolves size_t=0 (https://github.com/pantor/ruckig/blob/682dbafd45e41c94ed9d6ce45bb60b252e7faf14/include/ruckig/utils.hpp#L15). But the constructor has an argument that passes the dofs to the member, pretty sure these should be the number of joints

Reply:
thanks!

Reply:
The question really is that number of joints is not known at compile time